packages/fleather/example/lib/main.dart

```diff
@@ -17,6 +18,13 @@ class FleatherApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) => MaterialApp(
+        localizationsDelegates: const [
+          FleatherLocalizations.delegate,
+          GlobalCupertinoLocalizations.delegate,
+          GlobalMaterialLocalizations.delegate,
+          GlobalWidgetsLocalizations.delegate
+        ],
+        supportedLocales: const [Locale('en'), Locale('fr')],
```

Comment:
```suggestion
        supportedLocales: FleatherLocalizations.supportedLocales,
```

The list of locales is automatically generated.

Reply:
Thanks